test/challtestsrv/tlsalpnone.go

```diff
@@ -0,0 +1,114 @@
+package challtestsrv
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/sha256"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/letsencrypt/boulder/va"
+)
+
+// AddTLSALPNChallenge adds a new TLS-ALPN-01 key authorization for the given host
+func (s *ChallSrv) AddTLSALPNChallenge(host, content string) {
+	s.challMu.Lock()
+	defer s.challMu.Unlock()
+	s.taOne[host] = content
+}
+
+// DeleteTLSALPNChallenge deletes the key authorization for a given host
+func (s *ChallSrv) DeleteTLSALPNChallenge(host string) {
+	s.challMu.Lock()
+	defer s.challMu.Unlock()
+	if _, ok := s.taOne[host]; ok {
+		delete(s.taOne, host)
+	}
+}
+
+// GetTLSALPNChallenge returns the TLS-ALPN-01 key authorization for the given host
+// (if it exists) and a true bool. If the host does not exist then an empty
+// string and a false bool are returned.
```

Comment:
The "are returned" phrasing doesn't match the first sentence. Instead, how about "If the host does not exist, GetTLSALPNChallenge returns "", false"

In general I'd like to avoid "is returned" phrasing. It's passive voice and feels awkward to me.